config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-upgrade.yaml

```diff
@@ -91,7 +92,38 @@ periodics:
       - --skew
       - --test_args=--ginkgo.focus=\[Conformance\]|\[Feature:BootstrapTokens\]|\[Feature:NodeAuthorizer\] --minStartupPods=8
       - --timeout=300m
-      - --upgrade_args=--ginkgo.focus=\[Feature:MasterUpgrade\] --ginkgo.skip=statefulset-upgrade|hpa-upgrade|service-upgrade --upgrade-target=release/latest-1.9
+      - --upgrade_args=--ginkgo.focus=\[Feature:MasterUpgrade\] --ginkgo.skip=statefulset-upgrade|hpa-upgrade|service-upgrade --upgrade-target=release/latest-1.12
+
+- name: ci-kubernetes-e2e-kubeadm-gce-upgrade-1-12-1-13
```

Comment:
cc @neolit123 to double check if this looks correct

Reply:
the kubeadm changes LGTM.